bionic/debian/control

```diff
@@ -5,21 +5,21 @@ Section: science
 Priority: optional
 Build-Depends: debhelper (>= 11),
 	       libignition-cmake2-dev,
-  	       libignition-common3-dev,
-  	       libignition-fuel-tools4-dev,
-	       libignition-gazebo3-dev,
-	       libignition-gui3-dev,
+         libignition-common3-dev (>= 3.4),
+         libignition-fuel-tools4-dev (>=4.1),
```

Comment:
nit: whitespace is a little funny

Reply:
These files have such a mix of tabs and spaces that every time I try to fix one place, some other place gets messy. I think I fixed after you merged :D

Reply:
it's just a nit, so I merged, sorry if I wasted your time by mentioning it